### PR TITLE
:sparkles: find pool doesn't exist

### PR DESCRIPTION
**Pull request by MarioBatalha**
---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1206806202223953

### CODE REVIEW

File: views/pools/header/index.tsx [hunk 1]
```diff
@@ -1,19 +1,17 @@
 import { Box, Tabs, Typography } from '@interest-protocol/ui-kit';
-import { not } from 'ramda';
+//import { not } from 'ramda';
```
Comment (KipandaJr) [inline views/pools/header/index.tsx:2]: remove

File: views/pools/header/index.tsx [hunk 2]
```diff
@@ -1,19 +1,17 @@
...
+import FinPoolButton from './find-pool-button';
 import { HeaderProps } from './header.types';
-import SearchMobile from './search-mobile';
 
 const Header: FC<HeaderProps> = ({ currentTab, setTab }) => {
   const [showSearchField, setShowSearchField] = useState(false);
   const [isMobile, setIsMobile] = useState(false);
-
-  const handleCloseSearch = () => setShowSearchField(not);
+  //const handleCloseSearch = () => setShowSearchField(not);
```
Comment (KipandaJr) [inline views/pools/header/index.tsx:14]: remove

**Review by KipandaJr — CHANGES_REQUESTED** [review]
image
when i add other coin, the other r cleaned...

image
this button should be disabled if any field r empty

image
This modal has a diff BG that other (Find Pool eg.)

image
In mobile this modal r not aligned

File: views/pool-create/select-coins/input/select-token.tsx [hunk 3]
```diff
@@ -29,13 +29,14 @@ const SelectToken: FC<InputProps> = ({ index }) => {
...
-        value: '',
-      });
+    if (getValues('tokens')?.some((token) => token.type === type)) return;
+
+    setValue(`tokens.${index}`, {
+      type,
+      symbol,
+      decimals,
+      value: '',
+    });
```
[the remark this answers is not in the document]
Comment (MarioBatalha) [reply views/pool-create/select-coins/input/select-token.tsx:39]: It's not part of this task.

File: constants/fees.ts
[the remark this answers is not in the document]
Comment (MarioBatalha) [reply constants/fees.ts:1]: Give me a context of your comment

**Pull request merged**